app/src/main/java/org/simple/clinic/home/report/ReportsEvent.kt

```diff
@@ -1,8 +1,9 @@
 package org.simple.clinic.home.report
 
 import org.simple.clinic.util.Optional
+import org.simple.clinic.widgets.UiEvent
 
-sealed class ReportsEvent
+sealed class ReportsEvent : UiEvent
```

Comment:
We'll also need to override `analyticsName` on the events themselves in order to actually report them.

Reply:
I completely forgot about that 🤦🏾